ApiCompatTask/source/commandLineResult.ts

```diff
@@ -0,0 +1,48 @@
+import { TaskResult, getInput } from "azure-pipelines-task-lib";
+
+export default class CommandLineResult {
+    private _totalIssues: number;
+    private _body: string;
+
+    get totalIssues() {
+        return  this._totalIssues;
+    }
+
+    get body() {
+        return this._body;
+    }
+
+    constructor( result: string ) {
+        const indexOfTotalIssues = result.indexOf("Total Issues");
+        this._totalIssues = this.getTotalIssues(result, indexOfTotalIssues);
+        this._body = this.getBody(result, indexOfTotalIssues);
+    }
+
+    private getTotalIssues = (message: string, indexOfTotalIssues: number): number => {
+        return parseInt(message.substring(indexOfTotalIssues).split(':')[1].trim(), 10);
+    }
+    
+    private getBody = (message: string, indexOfTotalIssues: number): string => {
+        return message.substring(0, indexOfTotalIssues - 1);
+    }
+
+    public resultText() {
+        return this.totalIssues?

```

Comment:
Add space between totalIssues and the  ?. Right now is like this `this.totalIssues?` and it'd be `this.totalIssues ?`